common/app/services/dotcomrendering/OnwardsPicker.scala

```diff
@@ -1,4 +1,4 @@
-package services.dotcomponents
+package services.dotcomrendering
```

Comment:
I’m a fan of this change, but I’m not sure it should be included in this PR.

There’s also a lot of other places we would need to rename:
- [LiveBlogController.scala#L185-L186](https://github.com/guardian/frontend/blob/499856f7fd490ab200fd988cc20b4b856927362f/article/app/controllers/LiveBlogController.scala#L185-L186)
- [ArticlePicker.scala#L72-L78](https://github.com/guardian/frontend/blob/499856f7fd490ab200fd988cc20b4b856927362f/article/app/services/dotcomponents/ArticlePicker.scala#L72-L78)
- [DotcomRenderingService.scala#L93-L112](https://github.com/guardian/frontend/blob/499856f7fd490ab200fd988cc20b4b856927362f/common/app/renderers/DotcomRenderingService.scala#L93-L112)
- [DotcomponentsOnwardsModels.scala#L6](https://github.com/guardian/frontend/blob/499856f7fd490ab200fd988cc20b4b856927362f/onward/app/models/dotcomponents/DotcomponentsOnwardsModels.scala#L6)

Reply:
Agreed! This makes sense! I'll raise it straight after this gets merged

Reply:
Change made in this PR: https://github.com/guardian/frontend/pull/25506